### PR TITLE
Add support for Intermapping Cooperative Object Naming Standard

### PR DESCRIPTION
**Pull request by Mygod**
~~Blocked by https://github.com/Mygod/pokemon-icon-postprocessor/pull/1 but~~ can test it for now

### CODE REVIEW

File: src/views/index-js.mustache [hunk 1]
```diff
@@ -5169,47 +5169,25 @@ function getSize (size) {
...
+    for (const genderSuffix of genderSuffixes) {
+    for (const shinySuffix of shinySuffixes) {
+        const result = `${pokemonId}${evolutionSuffix}${formSuffix}${costumeSuffix}${genderSuffix}${shinySuffix}`;
+        if (availableForms.has(result)) return result;
+    }
+    }
+    }
+    }
+    }
+    return '0'; // substitute
```
Comment (versx) [inline src/views/index-js.mustache:5190]: 😂  Oh god the formatting

Comment (Mygod) [reply src/views/index-js.mustache:5190]: I think it looks nicer this way. :wink:

Comment (versx) [reply src/views/index-js.mustache:5190]: > I think it looks nicer this way. 😉

I actually do agree tbh, C# has something similar i.e:
```cs
using (var evolutionSuffix in evolutionSuffixes) {
using (var formSuffix in formSuffixes) {
using (var costumeSuffix in costumeSuffixes) {
using (var genderSuffix in genderSuffixes) {
using (var shinySuffix in shinySuffixes) {
    var result = $`{pokemonId}{evolutionSuffix}{formSuffix}{costumeSuffix}{genderSuffix}{shinySuffix}`;
    if (availableForms.Contains(result)) return result;
```

Comment (bschultz) [reply src/views/index-js.mustache:5190]: Please no 😜
Use indentation

Comment (Mygod) [reply src/views/index-js.mustache:5190]: I don't understand. They look identical.

Comment (versx) [reply src/views/index-js.mustache:5190]: Consistency though :P

Comment (bschultz) [reply src/views/index-js.mustache:5190]: @versx already added an eslint file, so please be consistent with the formatting rules

Comment (Mygod) [reply src/views/index-js.mustache:5190]: Maybe we should disable eslint on this function? I think an exception could/should be made in this case.

**Pull request merged**